src/olympia/editors/models.py

```diff
@@ -704,10 +706,46 @@ class AutoApprovalSummary(ModelBase):
     verdict = models.PositiveSmallIntegerField(
         choices=amo.AUTO_APPROVAL_VERDICT_CHOICES,
         default=amo.NOT_AUTO_APPROVED)
+    weight = models.IntegerField(default=0)
 
     def __unicode__(self):
         return u'%s %s' % (self.version.addon.name, self.version)
 
+    def calculate_weight(self):
+        """Calculate the weight value for this version according to various
+        risk factors.
+
+        That value is then used in editor tools to prioritize add-ons in the
+        auto-approved queue."""
+        # Note: for the moment, some factors are in direct contradiction with
+        # the rules determining whether or not an add-on can be auto-approved
+        # in the first place, but we'll relax those rules as we move towards
+        # post-review.
+        addon = self.version.addon
+        one_year_ago = (self.created or datetime.now()) - timedelta(days=365)
```

Comment:
`created` isn't nullable so I don't think the `or` is necessary or am I missing something?